.travis/docker-compose-travis.yml

```diff
@@ -27,3 +40,7 @@ services:
     depends_on:
       - "elasticsearch"
       - "mongo"
+
+volumes:
+  data01:
+    driver: local
```

Comment:
I also don't seem to need this `driver: local` line either, far as I can tell.

Reply:
It is need for travis